tests/tools.vitruv.change.correspondence.tests/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
Do we want to synchronize .settings too?

Reply:
We have to synchronize them to ensure that projects use Java 11 and UTF-8 encoding in Eclipse. Otherwise you have to remember setting that up manually everytime you create a new Eclipse workspace.